less-discourse/menu.css

```diff
@@ -0,0 +1,31 @@
+body {
```

Comment:
I was wondering what those files were in the first place...then you ask me why i deleted one? because i thought it was of no use :stuck_out_tongue_closed_eyes:  my bad!